src/content/en/updates/2020/08/devtools.md

```diff
@@ -0,0 +1,261 @@
+
+For engineers who are constantly testing video playback quality, working on a JavaScript front-end to a media player, or working on re-encoding tools, this new Media panel could greatly increase your productivity.
+
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bf98fea787a8c2db4e0763ae2095faf8112458eb #}
+
+Chromium issues: [1018414](https://crbug.com/1018414)
+
+
+## Issues tab updates {: #issues-tab }
+
+The Issues warning bar on the Console panel is now replaced with a regular message. 
+
+![Issues in console message](/web/updates/images/2020/08/issue-console-msg.png)
+
+Third party cookie issues are now hidden by default in the Issues tab. Tick the newly added “Include third-party cookie issues” checkbox to view them.
+
+![thrid party cookie issues checkbox](/web/updates/images/2020/08/third-party-cookies.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bb660b996589727edf16be103343f91d38b571fa #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/dea8897a72d4954ec04723bb0bbdf5c90440a0f1 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/760ee9c0d70d6bce108532c164fbdbc5cb2c2951 #}
+
+Chromium issues: [1096481](https://crbug.com/1096481), [1068116](https://crbug.com/1068116), [1080589](https://crbug.com/1080589)
+
+
+## Emulate missing local fonts {: #emulate-local-fonts }
+
+Open the [Rendering tab](/web/tools/chrome-devtools/evaluate-performance/reference#rendering) and use the new “Disable local fonts” feature to emulate missing `local() sources in @font-face rules.
+
+For example, when the font “Rubik” is installed on your device and the `@font-face src` rule uses it as a  `local()` font, Chrome uses the local font file from your device.
+
+When “Disable local fonts” is ticked, DevTools ignores the `local()` fonts and fetches them from the network.
+
+![Emulate missing local fonts](/web/updates/images/2020/08/disable-font.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/5b6a769ee19c084fdda6482d2d8f1ce9f7e37ac0  #}
+
+Chromium issues: [384968](https://crbug.com/384968)
+
+
+## Emulate inactive users {: #emulate-inactive-users }
+
+The [Idle Detection API](https://web.dev/idle-detection) allows developers to detect inactive users and react on idle state changes. You can now use DevTools to emulate idle state changes in the Sensors tab for both the user state and the screen state instead of waiting for the actual idle state to change.
+
+![Emulate inactive users](/web/updates/images/2020/08/emulate-idle.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0ab666a695bf08ffd479462c14e425f454b2b1a6  #}
+
+Chromium issue: [1090802](https://crbug.com/1090802)
+
+
+## Emulate `prefers-reduced-data` {: #emulate-prefers-reduced-data }
```

Comment:
FYI I historically have not documented experiments. But if you all want to get more feedback on the features, then it makes sense to document them. It's your call.

Reply:
There's some nuance here. To clarify, this is not a DevTools experiment (as in, you don't need to go to DevTools -> Settings -> Experiments to enable anything). The Web Platform feature itself is currently a Chrome experiment (behind a Chrome flag), but the DevTools feature is automatically available whenever the WP feature is supported. (In the future, when the feature ships by default, it will automatically become available in DevTools without the need for any flags to be enabled, without any code changes.)

As a user, IMHO it makes sense that if I enable a Chrome flag to get the WP feature, I can also test/debug it in DevTools.

Reply:
Whoops, I misread it. I thought it was a DevTools experiment, not a Chrome experiment. It's kind-of the same situation but your logic makes sense to me. Also, even if it is a DevTools experiment, if you all want to get feedback on the new feature, then it seems reasonable to cover it in What's New. I think we'll just have to keep in mind that we've never covered experiments before and may need to do some explaining about what they are and how they work.